src/yunohost/app.py

```diff
@@ -148,8 +148,12 @@ def app_list(offset=None, limit=None, filter=None, raw=False):
 
     for applist in applists:
         if '.json' in applist:
+            list_name=applist.split('.json', 1)[0]
             with open(repo_path +'/'+ applist) as json_list:
-                app_dict.update(json.loads(str(json_list.read())))
+                list_content=json.loads(str(json_list.read()))
+                for k in list_content.keys():
+                    list_content[k]['store']=list_name
```

Comment:
Exact, it was added because I push on my unstable branches. Github
automatically adds commit to the last PR not merged...

Those changes give the possibility to know from which list comes an
application (official, community, internetcube). It is important because
when a user add a list, he can't know if the app is official...

Le 14/02/2016 10:40, Jérôme Lebleu a écrit :
>
> In src/yunohost/app.py
> <https://github.com/YunoHost/yunohost/pull/122#discussion_r52840138>:
>
> >              with open(repo_path +'/'+ applist) as json_list:
> > -                app_dict.update(json.loads(str(json_list.read())))
> > +                list_content=json.loads(str(json_list.read()))
> > +                for k in list_content.keys():
> > +                    list_content[k]['store']=list_name
>
> What's the goal of this key? It seems not to be related to the other
> part, isn't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/YunoHost/yunohost/pull/122/files#r52840138>.
>